docs/tutorials/02a_training_a_quantum_model_on_a_real_dataset.ipynb

```diff
@@ -0,0 +1,891 @@
+    "svc.fit(train_features, train_labels)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "earned-destination",
+   "metadata": {},
+   "source": [
+    "Now we check out how well our classical model performs."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 8,
+   "id": "veterinary-proxy",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Classical SVC on the training dataset: 0.99\n",
+      "Classical SVC on the test dataset:     0.97\n"
+     ]
+    }
+   ],
+   "source": [
+    "train_score = svc.score(train_features, train_labels)\n",
+    "test_score = svc.score(test_features, test_labels)\n",
+    "\n",
+    "print(f\"Classical SVC on the training dataset: {train_score:.2f}\")\n",
+    "print(f\"Classical SVC on the test dataset:     {test_score:.2f}\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "limited-hybrid",
+   "metadata": {},
+   "source": [
+    "Well, SVC performs very well as expected from a well developed classical algorithm. Now it is time for quantum machine learning models.\n",
+    "\n",
+    "## 3. Training a Quantum Machine Learning Model\n",
+    "\n",
+    "As a quantum model we'll train a variational quantum classifier. This is the simplest available classifier in Qiskit Machine Learning and it can be a good starting point for the newcomers with a background in classical machine learning.\n",
+    "\n",
+    "But before we actually train a model, let's explore what `VQC` consists of. Two the most important parts are a feature map and an ansatz.\n",
+    "\n",
+    "Our data is classical. That means, we have our data as set of bits, not qubits. Apparently, we have to convert our representation from bits to qubits somehow. This is a very crucial process if we want to obtain a successful quantum model. This process is usually referred to as data encoding, data embedding or loading, or a feature map. This process does not appear in classical machine learning as we operate in the classical world only.\n",
+    "\n",
+    "Then, once the data is loaded we have to apply a parameterized quantum circuit straight after the loading part. This circuit is a direct analogue of the layers of classical neural networks. It has a set of tunable parameters or weights, and we must optimize the weights so that the model produces the expected results. Such quantum circuit is also called parameterized trial state, variational form or ansatz. Perhaps, the latter is the most widely used term.\n",
+    "\n",
+    "For more information we encourage the reader to take a look at the [Quantum Machine Learning Course](https://learn.qiskit.org/course/machine-learning).\n",
+    "\n",
+    "As a feature map we consider a ``ZZFeatureMap``. This is one of the feature maps available in Qiskit and makes use of entanglement. We pass `num_features` as `feature_dimenstion`, that means the feature map will have `num_features` or `4` qubits.\n",
```

Comment:
> If you are coming from classical and learning, this just drops `entanglement` on you with no explanation of anything. It may may use of it but how would that influence me in selecting/configuring a feature map for my dataset. We could even omit it from here and it would all still make sense. i.e. `This is one of the standard feature maps available in Qiskit circuit library.` But it may be worth saying something about it and data representation/spaces.

Replaced with your suggestion. I think for a sake of simplicity we can omit entanglement here. Otherwise we may end up with a very long text.

> 
> > We pass `num_features` as `feature_dimenstion`,
> 
> Spelling error in `feature_dimenstion`

Corrected the spelling error.